### ISSUE TITLE
Failed to set flags for new changesets since recently

### ISSUE THREAD

**Issue by Kovoschiz** [OSMCha/osmcha-frontend#716]
## *Brief Description*

Can't set flags on recent changesets. Can rate good/bad, and open the drop-down menu. Can change flags on older changesets. 

## What  is the current behaviour, (attach relevant screenshots) ?

Syntax error: JSON.parse: unexpected character at line 1 column 1 of the JSON data
HTTP 500 https://osmcha.org/api/v1/changesets/*/tags/*/

## What  is the expected behaviour ?

Should be able to set flags. Same usability on all changesets. 

## When does this occur ?

For all changesets after around 2024-02-19 UTC  (last I check)  

## How do we replicate the issue ?

1. Rate good/bad
2. Open the flags menu
3. Click on the flags

## Please tell us about your environment:

* Firefox 122.0 
   * Enhanced Tracking Protection off (I forgot how to unblock this one Google Analytics thing on basic protection, but it shouldn't matter?)
   * DuckDuckGo Privacy Essentials disabled
   * NoScript allowed all  
* Linux Mint 20.2

## Other Information / context:

Separate issue from #705 , which allowed me to set flags for some time. It's a new problem.

**Comment by willemarcel** [comment]
It should be working now. Thanks for reporting.
